src/components/link/link.spec.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import 'jest-styled-components';
+import TestRenderer from 'react-test-renderer';
+import { Link as RouterLink } from 'react-router';
+import Link from './link.component';
+import { assertStyleMatch } from '../../__spec_helper__/test-utils';
+
+const render = (props) => {
+  return TestRenderer.create(<Link { ...props }>test</Link>);
+};
+
+describe('Link', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
+    wrapper = shallow(<Link>test</Link>);
+  });
+
+  it('renders as expected', () => {
+    expect(render()).toMatchSnapshot();
+  });
+
+
+  it('if `disabled` prop has been passed to the component', () => {
```

Comment:
The wording for these `it` blocks is a little confusing, it might be better to have them say something like:
```
it('matches the expected style if `disabled` prop has been passed to the component', () => ...
```